src/status_im/native_module/core.cljs

```diff
@@ -1,130 +1,262 @@
 (ns status-im.native-module.core
-  (:require [status-im.native-module.impl.module :as native-module]))
+  (:require [clojure.string :as string]
+            [re-frame.core :as re-frame]
+            [status-im.react-native.js-dependencies :as rn-dependencies]
+            [status-im.ui.components.react :as react]
+            [status-im.utils.platform :as platform]
+            [status-im.utils.types :as types]))
+
+(defn status []
```

Comment:
why is it a function?

Reply:
I guess it is for tests

Reply:
might be worth moving it to status-im.react-native.js-dependencies at some point I'll add a comment